Qausim/epower_react_frontend
# Build the Header and Footer Components

#### What does this PR do?
Builds the header and footer components

#### How should this be manually tested?
Clone this repo, cd into it and 
- Run `npm start` and visit `localhost:3000` in a browser, the page should have a header and footer

#### Any background context you want to provide?
The app should have a header and a footer
